projects/plugins/data_providers/aime/aime4-bindaas-datasource-provider/src/main/java/edu/emory/cci/bindaas/datasource/provider/aime4/jaxb/EDDocRef.java

```diff
@@ -13,11 +13,12 @@
 import javax.xml.bind.annotation.XmlType;
 
 
+
 /**
```

Comment:
Pls don't change the styles of the classes in the pull request. It unnecessarily adds additional files to the review list, thus making the review a harder task.